test/controllers/qualtrics_controller_test.exs

```diff
@@ -3,9 +3,15 @@ defmodule ResearchResource.QualtricsControllerTest do
 
   setup %{conn: conn} = config do
     if username = config[:login_as] do
-      user = insert_user(email: username)
-      conn = assign(build_conn(), :current_user, user)
-      {:ok, conn: conn, user: user}
+      if username == "survey@completed.com"do
+        user = insert_user(%{email: username, qualtrics_id: "1"})
+        conn = assign(build_conn(), :current_user, user)
+        {:ok, conn: conn, user: user}
+      else
+        user = insert_user(email: username)
+        conn = assign(build_conn(), :current_user, user)
+        {:ok, conn: conn, user: user}
+      end
```

Comment:
My  code was similar but Elixir warn me to not use variable created in a scope outside of the scope:
![warning-user](https://cloud.githubusercontent.com/assets/6057298/24252898/b5b9062a-0fd6-11e7-8552-123164cc6476.png)

But agree I think there is a better way to do this.